src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -0,0 +1,124 @@
+package listeners;
+
+import io.qameta.allure.Attachment;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+import webdriver.DriverManager;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class OnlinerTestListener implements ITestListener {
```

Comment:
Please update name to 'TestListener'. What's the point in adding 'Onliner' to everything?